packages/office-ui-fabric-react/src/utilities/keytips/KeytipManager.ts

```diff
@@ -13,8 +14,9 @@ export interface IUniqueKeytip {
 export class KeytipManager {
   private static _instance: KeytipManager = new KeytipManager();
 
-  public keytips: IUniqueKeytip[] = [];
-  public persistedKeytips: IUniqueKeytip[] = [];
+  public keytips: { [Key: string]: IUniqueKeytip } = {};
+  public persistedKeytips: { [Key: string]: IUniqueKeytip } = {};
+  public sequenceMapping: { [Key: string]: IKeytipProps } = {};
```

Comment:
>sequenceMapping [](start = 9, length = 15)

what benefits does sequenceMapping give you?

Reply:
Similar reasoning for the refactoring of keytips. This is primarily used when we are looking for parent sequences of a keytips. In the past we would can through all keytips and then do an array comparison to see if we found the correct keytip.

With this change, we can do a instant lookup.